apps/lend/tsconfig.json

```
@@ -8,7 +8,7 @@
       "@/ui/*": ["../../../packages/ui/src/*"],
       "@/images": ["../../../packages/ui/src/images/index.ts"],
       "@/images/*": ["../../../packages/ui/src/images/*"],
-      "@/onboard": ["../../../packages/onboard-helpers/src/index.ts"],
+      "@/common/shared/features/*": ["../../../packages/curve-common/src/features/*"],
```

```suggestion
      "@/common/*": ["../../../packages/curve-common/src/*"],
```
I think that's enough